client/hooks/useIsScrolled.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useRef, useState } from '@wordpress/element';
+
+export default function useIsScrolled() {
+	const [ isScrolled, setIsScrolled ] = useState( false );
+	const rafHandle = useRef( null );
+	useEffect( () => {
+		const updateIsScrolled = () => {
+			setIsScrolled( window.pageYOffset > 20 );
+		};
+
+		const scrollListener = () => {
+			rafHandle.current = window.requestAnimationFrame(
+				updateIsScrolled
+			);
+		};
```

Comment:
I'm not sure whats going on, but the contents of the `scrollListener` don't get called on scroll so its not updating on scroll. The `is-scrolled` class is working well if I apply it manually though.

Reply:
Strange, it was working reliably for me on Friday. I'll take another look. You're just Firefox or Chrome?

Reply:
Yeah, can't reproduce this issue @psealock . Which page/etc are you on?

Reply:
Ok, it appears to be related to this error being displayed:

<img width="1027" alt="Screen Shot 2020-11-03 at 9 41 14 AM" src="https://user-images.githubusercontent.com/1922453/97917094-cfa7ca00-1db8-11eb-9c72-f03c84ff5bed.png">

When I comment out `define( 'WP_DEBUG', true );` to turn that off, it works great. I think the header uses some position CSS to go over the error message, but I have no idea why that would interfere with the scroll eventListener not being called.

Reply:
Ah, right. Not sure why it's conflict here either, but I've had my `WP_DEBUG` set to false since my page is overrun with deprecation warnings otherwise.